paasta_tools/utils.py

```diff
@@ -88,6 +87,7 @@
     "PAASTA_SYSTEM_CONFIG_DIR", "/etc/paasta/"
 )
 DEFAULT_SOA_DIR = service_configuration_lib.DEFAULT_SOA_DIR
+AUTO_SOACONFIG_SUBDIR = "auto"
```

Comment:
Has there been any discussion yet for making this a hidden dir, and also giving a more descriptive name, like `.automatically_applied_overrides`?

Reply:
I'm not against this being a .auto or .recommended_values or .reco
@solarkennedy they are not overrides because they only apply if a key is missing in the parent dir files. They aren't default either because a default value (which is a feature we still need to implement, typically for new services) is by definition not taylored/optimized for a given service.
Naming is hard

Reply:
`.automatically_tuned_settings`?

Reply:
I don't think we should make it a hidden directory. If it's possible that a developer would want to override something in them, I think they should be able to easily figure out their current settings. It's easy enough to find a setting like cpus from other sources, but for things like autoscaling, yelpsoa-configs would be the only place you could see that.

How about... `autotuned_default_configs`?

Reply:
I agree that making it a hidden directory will hurt  a developer's ability to find the value as @qui mentioned.

As for the name I agree with @matfra's opinion that the human values are the overrides so I like the theme of "automatically tuned" and "defaults". So +1 for @qui's suggestion of `autotuned_default_configs` or other similar name like `autotuned_defaults` or `automatically_tuned_defaults`

Reply:
I went with `autotuned_defaults`, thanks @ddelnano!